### PR TITLE
Switched up splashscreen for Flinstones theme

### PR DESCRIPTION
**Pull request by mdavidlawson**
Basic Screenshot. Leaves a lot to be desired but I wanted to get something up to represent our new theme.

Couple of problems:
  * Images don't size well right now, they'll fit horozontally but not vertically. 
  * I didn't use the vectored version of the image, I'd like to take a closer look at that before we put it in. Couple of things I'd like to understand first.
  * There's some cleanup needed in the splash scss file and the main scss file but we can come back to that later

### CODE REVIEW

File: htdocs/sites/mayofest15_dev/app/styles/vendor.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@ $icon-font-path: "/bower_components/bootstrap-sass-official/vendor/assets/fonts/
 
 @import "_variables.scss";
 // bower:scss
-@import "../bower_components/bootstrap-sass-official/vendor/assets/stylesheets/bootstrap.scss";
```
Comment (kheaactua) [inline htdocs/sites/mayofest15_dev/app/styles/vendor.scss]: Just a comment, there's something about the `bower_components` path I've never understood, it keeps getting broken automatically.  You might encounter this.  I typically revert this one change back here.

File: htdocs/sites/mayofest15_dev/app/views/splash.html [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
 						title="Regi-Star (sorry)" ui-sref="home.login">
 						Register<span class="hidden-xs"> / Login</span>
 					</button>
-					<a class="btn btn-primary btn-primary-starwars"
-						title="What is Moktoberfest?!"
-						href="../moktoberfest14/">
-						Moktoberfest!
+					<a class="btn btn-primary btn-primary-flintstone"
+						title="What is Mayofest?"
+						href="../mayofest15_dev">
```
Comment (kheaactua) [inline htdocs/sites/mayofest15_dev/app/views/splash.html]: We'll have to make sure this link knows when to go to the dev or real site.

**Comment by kheaactua** [comment]
Looks good, but yeah, we gotta switch to the SVG, or at least the rasterized SVG. :)

**Comment by mdavidlawson** [comment]
Agreed. just after we can take ppl's money

**Pull request merged**